dashboard/client/src/pages/job/index.tsx

```diff
@@ -110,7 +110,9 @@ const JobList = () => {
                         {isDead ? "true" : "false"}
                       </TableCell>
                       <TableCell align="center">
-                        {dayjs(timestamp * 1000).format("YYYY/MM/DD HH:mm:ss")}
+                        {dayjs(Math.floor(timestamp)).format(
```

Comment:
Good catch! Isn't timestamp already an integer?

Reply:
Oh, my bad, it's a string.

Reply:
nit: can we then actually use `Number(timestamp)` to explicitly convert from string to integer since we don't need to do a `floor` here.

Reply:
Updated :+1: